server/tserver/src/main/java/org/apache/accumulo/tserver/session/ScanSession.java

```diff
@@ -180,9 +183,18 @@ public TabletResolver getTabletResolver() {
     return tabletResolver;
   }
 
+  public ScanTask<T> getScanTask() {
+    return scanTask;
+  }
+
+  public void setScanTask(ScanTask<T> scanTask) {
+    this.scanTask = scanTask;
+  }
+
   @Override
   public boolean cleanup() {
     tabletResolver.close();
+
```

Comment:
The cleanup method could be simplified:
```suggestion
  @Override
  public boolean cleanup() {
    tabletResolver.close();
    return super.cleanup();
  }
```